modules/nf-core/krakentools/extractkrakenreads/meta.yml

```diff
@@ -0,0 +1,53 @@
+name: "krakentools_extractkrakenreads"
+description: Extract reads classified at any user-specified taxonomy IDs.
+keywords:
+  - kraken
+  - krakentools
+  - metagenomics
+tools:
+  - "krakentools":
+      description: "KrakenTools is a suite of scripts to be used for post-analysis of Kraken/KrakenUniq/Kraken2/Bracken results. Please cite the relevant paper if using KrakenTools with any of the listed programs."
+      homepage: https://github.com/jenniferlu717/KrakenTools
+      documentation: https://github.com/jenniferlu717/KrakenTools?tab=readme-ov-file#extract_kraken_readspy
+      tool_dev_url: https://github.com/jenniferlu717/KrakenTools
+      doi: 10.1038/s41596-022-00738-y
+      licence: ["GPL v3"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - classified_reads_assignment:
+      type: file
+      description: A file contains the taxonomic classification of each input read.
+      pattern: "*{classifiedreads.txt}"
+  - classified_reads_fastq:
+      type: file
+      description: Classified reads as belonging to any of the taxa on the kraken2 database.
+      pattern: "*{fastq.gz}"
+  - taxid:
+      type: string
+      description: A list of taxid separated by spaces
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - extracted_kraken_reads:
+      type: file
+      description: Reads assigned to a taxid list.
+      pattern: "*.{fastq}"
```

Comment:
Same here, should the output be `*.{fasta,fa}`

Reply:
`--fastq-output` was included in the script so that the output files are in the FASTQ format. This allows us to perform quality control (QC) checks and run mapping with Bowtie2.

Reply:
You mean that it should be `"*.{fastq, fasta, fa}"` ?

Reply:
Yes you are right!

Reply:
Both `fastq` and `fasta` were included in output formats.